data/data-pipeline/data_pipeline/etl/score/etl_score_post.py

```diff
@@ -1,6 +1,7 @@
 import json
 import zipfile
 from pathlib import Path
+from dynaconf.loaders.py_loader import try_to_load_from_py_module_name
```

Comment:
oh actually i dont know if this needs to be here / why i added it. maybe its leftover from a merge conflict.